.github/workflows/publish-galaxy.yml

```diff
@@ -20,25 +20,6 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Build and publish
-        shell: bash
-        run: |
-          # Ensure ref name looks like semver
-          ref_name="${{ github.ref_name }}"
-          if [[ ! "$ref_name" =~ ^v([0-9]+\.[0-9]+\.[0-9]+) ]]; then
-            echo "Ref name '${ref_name}' is not of the form 'vX.Y.Z'"
-            exit 1
-          fi
-
-          # Ensure version nonempty
-          version="${BASH_REMATCH[1]}"
-          if [[ -z "$version" ]]; then
-            echo "Empty version '${version}'"
-            exit 1
-          fi
-
-          # Build the Collection tarball
-          ansible-galaxy collection build
-
-          # Publish the Collection
-          ansible-galaxy collection publish digitalocean-cloud-${version}.tar.gz \
-            --token ${{ secrets.GALAXY_TOKEN }}
+        uses: artis3n/ansible_galaxy_collection@v2
```

Comment:
:)

Reply:
Replacing for simplicity of this file, and we perform the same semver check:

https://github.com/artis3n/ansible_galaxy_collection/blob/main/src/Collection.ts#L18-L20

https://github.com/artis3n/ansible_galaxy_collection/blob/main/src/decorators.ts#L2